frontend/app/src/config/host.ts

```diff
@@ -3,7 +3,7 @@ const externalDockerHosts = ['localhost:23007', 'localhost:23000'];
 
 export function getHost(): string {
   const host = window.location.host.includes('localhost') ? 'localhost:5002' : window.location.host;
-  return host;
+  return 'people.sphinx.chat';

```

Comment:
please change this back, this will also effect the unit tests

Reply:
This is also for another issue, I'll create a different PR for it